### DISCUSSION TITLE
Trilium hosting

### DISCUSSION THREAD

**Discussion by maphew** [TriliumNext/Trilium#6048] [General]
As of october 2024 the only advertised place for managed Trilium hosting is https://trilium.cc. I've signed up for a year there and my experience 1 week in is quite positive. The performance while roaming is noticeably better than my prior home setup on NAS. Getting started was painless, excepting a small hiccup with the Stripe payment interstitial page that appeared to not complete but actually did.

Trilium.cc is running the original zadam Trilium Notes. I asked the owner, @sigaloid, if he plans to migrate to Trilium Next: no, not at present since they're not appreciably different on the backend. He's committed to running and maintaining trilium.cc indefinitely, but is close to the limit of what can be managed satisfactorily. He pointed me to **PikaPods** as being a possible future hosting provider. I've since corresponded with PP tech support about adding Trilium Next. They said it tooked straigthforward and they will add it to their app list -- no commitment to when however, a when they have time kind of deal.

Sooo, **hey everybody! let Piki Pod know Trilium Next is a desirable app for their platform at https://feedback.pikapods.com/posts/134/trilium-personal-knowledge-base**

**Comment by austin-barrington** [comment]
Hey @maphew, I'd be up for setting up a dedicated triliumNext hosting platform. If you think this would be preferred over using the pikipod provider. I've got a history of gameserver hosting and would be excited to chat about how I could help.

**Reply by maphew** [reply]
I've no specific opinion on PikaPods other than: their website looks good, the price is reasonable, and their promise of sharing of revenue with the upstream projects is a super +1. However at this point since Trilium is not available with them it's just words. : - )

As for you setting up a hosting business, go for it! There's no reason for there to be only one. I advise thinking carefully about handling properly though. Personal Knowledge Management systems, of which Trilium is one, are all about people collecting their valuable thought things in one basket.

**Reply by austin-barrington** [reply]
Yes absolutely personal knowledge management is important. Running a platform for this would need to ensure security, availablity and quality backups 😅. 

I've got experience in running global scale systems that this would be alright to manage. I was thinking about revenue share and supporting where I can in exchange for endorsement of some kind.

I'd be happy to work with you and other maintainers to ensure it's done right.

**Reply by maphew** [reply]
That's good to hear Austin! I should make it clear: I make noise and am a member of the organisation but am not a maintainer, just someone who's only recently chosen Trilium to be my next and hopefully last PKM home base. (meaning: I can't say anything on a possible endorsements front)

My personal reason for starting this thread is frustration with my inability to reliably self-host and not finding many choices for managed hosting, and none for Trilium Next.

**Reply by austin-barrington** [reply]
That's totally cool Matt, thanks. I was on the lookout for a fully open source knowledge management system which got me to here. Figured if I'm going to commit to one system then I'd like to assist in helping others with it. Your post got me thinking this is how I could assist.

**Comment by maphew** [comment]
folks who thumbs-down, please explain why. If there's something about PikaPods that does not make them a good provider I'd like to know

**Comment by maphew** [comment]
>"This app is now available to run on PikaPods: https://www.pikapods.com/pods?run=trilium-next
> It's marked as "experimental" since it's new and seems to develop quickly. Would appreciate your feedback regarding any deployment bugs or missing settings we may be able to address on our side.
> -- Pika Pods Support
